### PR TITLE
normalize properly async functions

### PR DESCRIPTION
**Pull request by cblegare**

### CODE REVIEW

**Comment by agronholm** [comment]
Please explain this PR. The test tests that `normalize_source_lines()` returns exactly what was given to it. What's the point?

**Comment by agronholm** [comment]
I just read the linked issue and it shed some light into this.

File: tests/test_sphinx_autodoc_typehints.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 from sphinx_autodoc_typehints import (
     format_annotation, get_annotation_args, get_annotation_class_name, get_annotation_module,
-    process_docstring)
+    process_docstring, normalize_source_lines)
```
Comment (agronholm) [inline tests/test_sphinx_autodoc_typehints.py:15]:
```suggestion
    normalize_source_lines, process_docstring)
```

**Comment by cblegare** [comment]
Hi @agronholm, thank you for reviewing this.  I see there are some failed checks, and I am not sure how you prefer to handle this next.

Feel free to suggest more changes, I'll be glad to implement them.

**Comment by agronholm** [comment]
The failing tests are not caused by this PR. They're caused by Sphinx's behavioral changes in recent updates.

**Comment by agronholm** [comment]
Thanks!

**Pull request merged**